en/getting_started/flight_reporting.md

```diff
@@ -6,13 +6,34 @@ PX4 logs detailed aircraft performance data once the system has been armed until
 
 ## Downloading Logs from the Flight Controller
 
-Logs can be downloaded using [QGroundControl](http://qgroundcontrol.com/) (v3.2 or later): **Analyze View > Log Download**.
+Logs can be downloaded using [QGroundControl](http://qgroundcontrol.com/) (v3.2 or later): **[Analyze View > Log Download](https://docs.qgroundcontrol.com/en/analyze_view/log_download.html)**.
 
-![Flight Log Download](../../images/flight_log_download.png)
+![Flight Log Download](../../assets/qgc/analyze/log_download.jpg)
 
 
 ## Sharing the Log Files for Review by PX4 Developers
 
-Upload the log files to [http://review.px4.io](http://review.px4.io). The log file link can then be shared for review in [discussion forums](../README.md#support) or a [Github issue](../README.md#reporting-bugs--issues).
+Upload the log files to [Flight Review](http://logs.px4.io) (http://logs.px4.io). 
+The log file link can then be shared for review in [discussion forums](../README.md#support) or a [Github issue](../README.md#reporting-bugs--issues).
+
+> **Tip** The topic [Flight Log Analysis](https://dev.px4.io/en/log/flight_log_analysis.html) (PX4 Developer Guide) contains basic information about log file analysis (and alternative analysis tools).
+
+
+## Additional Configuration
+
+The logging system is configured by default to collect sensible logs for use with [Flight Review](http://logs.px4.io).
+
+Logging may further be configured using the [SD Logging](../advanced_config/parameter_reference.md#sd-logging) parameters.
+The parameters you are most likely to change are listed below.
+
+Parameter | Description
+--- | ---
+[SDLOG_MODE](../advanced_config/parameter_reference.md#SDLOG_MODE) | Logging Mode defines when logging starts and stops.<br />- `0`: log when armed until disarm (default)<br />- `1`: log from boot until disarm<br />- `2`: log from boot until shutdown.
+[SDLOG_PROFILE](../advanced_config/parameter_reference.md#SDLOG_PROFILE) | Logging profile. Use this to enable less common logging/analysis (e.g. for EKF2 replay, high rate logging for PID & filter tuning, thermal temperature calibration).
+[SDLOG_MISSION](../advanced_config/parameter_reference.md#SDLOG_MISSION) | Create very small additional "Mission Log". 
+  This log can *not* be used with *Flight Review*, but is useful when you need a small log for geotagging or regulatory compliance.
```

Comment:
I might add some upload protection for flight review to avoid mission logs from being uploaded. On the other hand, it might be useful as you can still see the flown trajectory and metadata.

Reply:
OK. I'm going to merge this now though, because it is true "for now".